README.md

```diff
@@ -13,7 +13,11 @@ You gain access to the same public statistical information that is on the websit
 
 ## Install
 ```bash
+# with npm
 npm install --save socialblade
+
+# with yarn
+yarn add socialblade
```

Comment:
I think the additional clarity here is wonderful! However, do you think it'd be better to have two separate code blocks instead of keeping them in the same?
Similar to fastify: https://github.com/fastify/fastify#install

Reply:
I think so we should keep it the same instead of keeping two different code blocks!